tests/unit/components/DocumentationTopic/DocumentationNav.spec.js

```diff
@@ -59,6 +61,9 @@ describe('DocumentationNav', () => {
     currentTopicTags: [{
       type: 'foo',
     }],
+    interfaceLanguage: 'swift',
+    swiftPath: 'documentation/foo',
+    objcPath: 'documentation/foo',
```

Comment:
Just so its clear they are different.

```suggestion
    objcPath: 'documentation/bar',
```